ez_sphinx/bar/hmm.py

```diff
@@ -11,17 +11,31 @@ class HMM:
 	def __init__(self, study_time=25):
 		self.study_time= study_time
 		self.is_studying = False
-
+		self.friends = ['lgg','jgg','kxg','zzl']
 	def study(self):
 		"""HMM starts to study."""
 		self.is_studying = True
 		for i in range(self.study_time):
 			print(f"is studing: {self.is_studying}")
 
-	def get_study_state(self):
+	@property
+	def is_studying(self):
 		"""Get the state indicating if HMM is currently studying.
 
 		Returns:
 			bool: if True, HMM is currently studying. Otherwise, she must be playing with the mobile phone.
 		"""
 		return self.is_studying
+	def offer_coffee(self, people):
```

Comment:
1. Rename this method. E.g. `allow_coffee`.
2. Change `people` to `person`.

Reply:
[。。。](https://www.google.com/url?sa=i&url=http%3A%2F%2Fwww.51yuansu.com%2Fsc%2Flqrfywaiuf.html&psig=AOvVaw0xjAbN5Ba2N14ZmEsPcjPx&ust=1626088431479000&source=images&cd=vfe&ved=0CAoQjRxqFwoTCNDEhuTW2vECFQAAAAAdAAAAABAD)

Reply:
> 1. Rename this method. E.g. `allow_coffee`.
> 2. Change `people` to `person`.

Done!

Reply:
@zzhuolun. push the update to your branch.

Reply:
@zzhuolun. Also add docs for the input `person`

Reply:
please check the new Pull Request